File: Web_search_job/DatabaseClasses/UserFolder/Entities/GetData/UserData.cs
using Web_search_job.DatabaseClasses.FiltersFolder;

namespace Web_search_job.DatabaseClasses.UserFolder.Entities.GetData
{
    public class UserData
    {
        public string? Id { get; set; } = null!;
        public string? Email { get; set; } = null!;
        public string? FirstName { get; set; } = null!;
        public string? LastName { get; set; } = null!;
        public int? UserAge { get; set; }
        public string? UserImg { get; set; } = null!;
        public Location? Location { get; set; } = null!;
        public string? NumberPhone { get; set; } = null!;
        public string? Gender { get; set; } // Додана властивість для дати народження
    }
}

File: Web_search_job/ClientApp/src/app/models/backend/dtos/auth/dtos/user-data.ts
import {Location} from "../../other/location.dto";
import {LocationDataDTO} from "../../other/location-data.dto";

export interface UserData {
  id: number;
  userId: string;
  email: string;
  firstName: string;
  lastName: string;
  userAge: number;
  userImg: string;
  dateOfBirth: Date;
  location: LocationDataDTO;
  phoneNumber: string;
}
